### DISCUSSION TITLE
Documentation for v3

### DISCUSSION THREAD

**Discussion by pixobit** [nette/latte#296] [Q&A] [answered]
Where can I find the documentation for v3, and the changes that I need to look out for migrating from v2?

**Comment by dg** [comment]
I'm still writing it.

**Comment by dg** [comment, marked as answer]
Doc is ready https://latte.nette.org/en/cookbook/migration-from-latte2

**Comment by MartinMystikJonas** [comment]
@dg May I suggest to add docs with explanation how pair tags definition in Latte 3 works. I am currrently working on Latte 3 support for contributte/translation and I am bit lost in that Generator/yield magic :-)

Also is there some easy way to convert ArrayNode with dynamic parameters to list of parameters in function call. Basically instead of formating it as `[param1, param2]` format it as `param1, param2`. So far I found it is implemented in FilterNode but how can I use it in custom function call?
